<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Login Test"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0,009">
  <properties />
            <testcase name="loginTest" time="0,009" classname="testCases.LoginTest">
                <failure type="java.lang.IllegalArgumentException"
                  message="Cannot find elements when the XPath is null."
              >
<![CDATA[java.lang.IllegalArgumentException: Cannot find elements when the XPath is null.
  org.openqa.selenium.By$ByXPath.<init>(By.java:339)
  org.openqa.selenium.By.xpath(By.java:92)
  base.Page.click(Page.java:198)
  pages.HomePage.goToLogin(HomePage.java:21)
  testCases.LoginTest.loginTest(LoginTest.java:19)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:566)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
  org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
  org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
  org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
  org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
  org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
  org.testng.TestRunner.privateRun(TestRunner.java:794)
  org.testng.TestRunner.run(TestRunner.java:596)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
  org.testng.SuiteRunner.run(SuiteRunner.java:276)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
  org.testng.TestNG.runSuites(TestNG.java:1063)
  org.testng.TestNG.run(TestNG.java:1031)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
